src/firebase/auth/functions/config/emailTemplates.js

```diff
@@ -1,94 +1,47 @@
+/* eslint-disable indent */
 /* eslint-disable max-len */
 
 require("dotenv").config();
 const baseUrl = process.env.PLATFORM_BASE_URL;
 
 module.exports = {
-  mobileAppWelcome: (email, name) => {
```

Comment:
Minor refactoring to reduce code repetition. Similar refactoring was done in the emailTemplates file in auth Service a while back.